ratpack-cassandra/src/main/java/smartthings/ratpack/cassandra/CassandraModule.java

```diff
@@ -139,8 +161,93 @@ public void setPassword(String password) {
 
 	@Override
 	protected void configure() {
+		bind(Session.class).to(DefaultSession.class).in(Scopes.SINGLETON);
```

Comment:
But I guess that would require you to move the Session creation logic into some sort of factory service that could be overridden.  Which maybe then it's more or less equivalent.